learning_resources/models.py

```diff
@@ -980,6 +980,7 @@ class PodcastEpisode(LearningResourceDetailModel):
     )
 
     transcript = models.TextField(blank=True, default="")
+    audio_url = models.URLField(null=True, max_length=2048)  # noqa: DJ001
```

Comment:
Could we make this `null=False`?

Both of these things indicate it's required:
- https://github.com/Podcast-Standards-Project/PSP-1-Podcast-RSS-Specification?tab=readme-ov-file#required-item-elements
- https://podcasters.apple.com/support/823-podcast-requirements 
    > Each episode must have a unique [<enclosure>](https://help.apple.com/itc/podcasts_connect/#/itcb54353390) tag, along with its three required components — URL, length, and type ...